public/app/plugins/datasource/mssql/partials/config.html

```diff
@@ -56,6 +62,32 @@ <h3 class="page-heading">MS SQL connection</h3>
 			</info-popover>
 		</div>
 	</div>
+
+  <div class="gf-form" ng-show="ctrl.showTlsConfig">
+    <gf-form-switch class="gf-form" label="Skip TLS/SSL Verify" label-class="width-15"
+                      tooltip="Skip verifying Server Certificate for TLS/SSL. Needed when using self signed certificates."
+                      checked="ctrl.current.jsonData.tlsSkipVerify" switch-class="max-width-8"></gf-form-switch>
+  </div>
+
+  <div class="gf-form max-width-30" ng-show="ctrl.showTlsConfig">
+    <span class="gf-form-label width-15">TLS/SSL Root Certificate</span>
+    <input type="text" class="gf-form-input" style="width: 352px" ng-model='ctrl.current.jsonData.certificate'
+           placeholder="TLS/SSL root certificate file"></input>
+    <info-popover mode="right-absolute">
+      Path to file containing the public key certificate of the CA that signed the SQL Server certificate. Needed when
+      the server certificate is self signed, and <i>Skip TLS/SSL Verify</i> is <i>false</i>.
+    </info-popover>
+  </div>
+
+  <div class="gf-form max-width-30" ng-show="ctrl.showTlsConfig">
+    <span class="gf-form-label width-15">Hostname in server certificate</span>
+    <input type="text" class="gf-form-input" style="width: 352px" ng-model='ctrl.current.jsonData.hostNameInCertificate'
+           placeholder="Common Name (CN) in server certificate"></input>
+    <info-popover mode="right-absolute">
+      Specifies the Common Name (CN) in the server certificate. Default is the server host.
+    </info-popover>
+  </div>
```

Comment:
Currently I see this which makes me a bit confused 
![image](https://user-images.githubusercontent.com/1668778/119692816-04aa6580-be4c-11eb-804b-fa029f7043e8.png)

If you hide these fields when SKIP TLS/SSL Verify is false it will make a better user experience and you can perhaps skip the tooltip help text what's needed and what's not?